sdk/openai/Azure.AI.OpenAI/src/Generated/AzureCognitiveSearchChatExtensionConfiguration.cs

```diff
@@ -16,27 +16,6 @@ namespace Azure.AI.OpenAI
     /// </summary>
     public partial class AzureCognitiveSearchChatExtensionConfiguration
     {
-        /// <summary> Initializes a new instance of AzureCognitiveSearchChatExtensionConfiguration. </summary>
-        /// <param name="type">
-        /// The type label to use when configuring Azure OpenAI chat extensions. This should typically not be changed from its
-        /// default value for Azure Cognitive Search.
-        /// </param>
-        /// <param name="searchEndpoint"> The absolute endpoint path for the Azure Cognitive Search resource to use. </param>
-        /// <param name="searchKey"> The API admin key to use with the specified Azure Cognitive Search endpoint. </param>
-        /// <param name="indexName"> The name of the index to use as available in the referenced Azure Cognitive Search resource. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="searchEndpoint"/>, <paramref name="searchKey"/> or <paramref name="indexName"/> is null. </exception>
-        internal AzureCognitiveSearchChatExtensionConfiguration(AzureChatExtensionType type, Uri searchEndpoint, string searchKey, string indexName)
-        {
-            Argument.AssertNotNull(searchEndpoint, nameof(searchEndpoint));
-            Argument.AssertNotNull(searchKey, nameof(searchKey));
-            Argument.AssertNotNull(indexName, nameof(indexName));
-
-            Type = type;
-            SearchEndpoint = searchEndpoint;
-            SearchKey = searchKey;
```

Comment:
I saw you change `SearchKey` to string now. If that is because the limitation of `AzureKey`, let me know, we have solution. But if it is expected, just ignore this.

Reply:
This is expected. Thanks!